lib/components/molecules/QuoteMedia/QuoteMedia.story.js

```diff
@@ -1,25 +1,24 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 // import { action } from '@storybook/addon-actions';
-import defaultConfig from './QuoteMedia.mock';
 import Image from '../../atoms/Image/Image';
 // Import Styled Component to showcase variations
 import QuoteMedia, { QuoteMediaVanilla } from '.';
-
-// TODO: indentation is not working fine in editor. Need to check for .hbs files
+import defaultImage from '../../atoms/Image/Image.mock';
+import { defaultQuoteMedia, stackedQuoteMedia } from './QuoteMedia.mock';
 
 storiesOf('Molecules/QuoteMedia', module)
   .addParameters({
     jest: ['QuoteMedia', 'QuoteMediaVanilla'],
   })
-  .add('Knobs', () => (
-    <QuoteMediaVanilla
-      {...defaultConfig}
-      className={`hide-default-sample ${defaultConfig.className}`}
-    />
+  .add('Knobs', () => <QuoteMediaVanilla {...defaultQuoteMedia} />)
+  .add('Column', () => (
+    <QuoteMedia {...defaultQuoteMedia}>
+      <Image {...defaultImage} />
+    </QuoteMedia>
   ))
-  .add('Default', () => (
-    <QuoteMedia {...defaultConfig}>
-      <Image />
+  .add('Stacked', () => (
+    <QuoteMedia {...stackedQuoteMedia}>
+      <Image {...defaultImage} />
     </QuoteMedia>
   ));
```

Comment:
We should define two variations column and stacked where column is default variation.